src/Api/Endpoints/Candidate/Visit/Qc.php

```diff
@@ -0,0 +1,207 @@
+<?php declare(strict_types=1);
+/**
```

Comment:
Can you please make sure to integrate the changes from this PR that got merge for the imaging QC information?
https://github.com/aces/Loris/pull/4077